localhost

```diff
@@ -1 +1 @@
-localhost
+localhost ansible_python_interpreter=/usr/local/bin/python2.7
```

Comment:
Ah, it does work without this -- you just have to run ansible-pull in the repository directory.